examples/11_melt_native_token.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example 11_melt_native_token --release
+// In this example we will melt an existing native token with its foundry
+// Rename `.env.example` to `.env` first
+
+use std::{env, str::FromStr};
+
+use dotenv::dotenv;
+use iota_client::bee_block::output::TokenId;
+use iota_wallet::{account_manager::AccountManager, Result, U256};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // This example uses dotenv, which is not safe for use in production
+    dotenv().ok();
+
+    // Create the account manager
+    let manager = AccountManager::builder().finish().await?;
+
+    // Get the account we generated with `01_create_wallet`
```

Comment:
Just for understanding, this example assumes that the other example `01_create_wallet` was started before right?

Reply:
Yes, this is actually required for all examples